plugins/modules/mssql_script.py

```diff
@@ -165,6 +191,27 @@
 """
 
 RETURN = r"""
+script_source:
+  description: Source of the executed script.
+  type: str
+  returned: always
+  sample: "file"
+  choices: ["file", "parameter"]
+script_path:
+  description: Path to the script file that was executed.
+  type: str
+  returned: when script_path parameter is used
+  sample: "/path/to/script.sql"
+bom_removed:
+  description: Whether a Byte Order Mark was detected and removed from the script file.
+  type: bool
+  returned: when script_path parameter is used and BOM was found
```

Comment:
Why not always provide this if `script_path` is used? It's not very helpful if this is sometimes `true` and sometimes not there.